R/select_datasets.R

```diff
@@ -0,0 +1,22 @@
+select_europages_companies <- function(data) {
+  data |>
+    select(
+      "company_name",
+      "country",
+      "company_city",
+      "postcode",
+      "address",
+      "main_activity",
+      "companies_id"
+    )
+}
+
+select_ecoinvent_inputs <- function(data) {
+  data |>
+    select(
+      "input_activity_uuid_product_uuid",
+      "exchange_name",
+      "exchange_unit_name"
+    ) |>
+    distinct()
```

Comment:
@kalashsinghal do we need `distinct()` above too?

Reply:
@maurolepore There could be multiple similar `input_activity_uuid_product_uuid` in the original `ecoinvent_inputs` file from tiltIndicatorBefore, hence we need `distinct()` above to remove those duplicates. 

I can select these columns and then remove the duplicates in tiltIndicatorBefore as well, but I think this should happen in tiltIndicatorAfter as this processing is requirement for tiltIndicatorAfter. Do you agree?

Reply:
Thanks!
I follow up at https://github.com/2DegreesInvesting/tiltIndicatorAfter/pull/107